### PR TITLE
First Code Review

### PR DESCRIPTION
**Pull request by David405**

### CODE REVIEW

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Comment by Torres-ssf** [comment]
### Nice work @David405 
You worked hard on this project but before merging to master we need to do some changes in order to make it in line with Microverse guidelines. They are:
- [ ] Your navbar should be made of 9 nine items, just like the original page. Kindly add the ones that are missing.

- [ ] The search field should be an `input` HTML tag with the focus effect enabled

- [ ] Its best practice to use the main tag for the most important content on your page. In this project's case, it the 3 main sentences.

- [ ] All the 3 main sentences must be a link. They also need to be loaded as images(they are images in the original page). You must load them using a div tag for each one. Use the background from the div tag to load the images. Just like the 'hero' image. Since each one of the sentences must be a link, you will have place each div inside an anchor tag( 3 `a` tag and 3 `div` tag total).

- [ ] Use footer tag instead of div tag for the footer.

- [ ] Microverse guidelines require that the 4 promo images must be part of the footer. The footer must be divided into 2 parts, top, and bottom. The top part must contain the 4 promo images.

- [ ] The 4 promo images must use the same approach of the 3 main sentences. Use div tags for each one and make each one a link. Load the images using background from the div tag.

- [ ] The appearance of your page does not match the style from the original one. Try to match the same style as much as possible.
-Original page:
image

- [ ]  There are some errors in your HTML code. Check them using this HTML [validator](https://validator.w3.org/) . Ignore the warnings.

Kindly make these changes and submit for code review again. If you are struggling with any of those changes feel free to message me on Slack. I will help you.

**Comment by David405** [comment]
@Torres-ssf I didn't get a notification from this pull request earlier, I just saw your comment today and have fixed most of the errors you pointed out. Thank you for your time.

**Comment by Ispirett** [comment]
#Reviewing

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<html>
+    <head>
+        <link href="style.css" rel="stylesheet">
+        <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+        <link href="https://fonts.googleapis.com/css?family=Cormorant&display=swap" rel="stylesheet">
+
+        <title>Apple</title>
+    </head>
+    <body>
+        <div class="container">
```
Comment (Ispirett) [inline index.html]: This `<div>` is not closed.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<html>
```
Comment (Ispirett) [inline index.html]: In your <Html > i'ts wise to specify the language example lang="en"

**Comment by David405** [comment]
Hello TSE, I have fixed the errors found by HTML validator service. I have requested for a final code review.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```
Comment (bolah2009) [inline index.html]: `SUGGESTION`

Consider adding `lang` attribute.

Comment (David405) [reply index.html]: I made changes as requested @bolah2009

File: style.css [hunk 4]
```diff
@@ -2,7 +2,7 @@ body {
   font-family: 'Cormorant', serif;
 }
 .container {
-    width: 76.5%;
+    width: 76.50%;
     text-align: center;
     margin-left:  200px;
```
Comment (Oghenebrume50) [inline style.css]: Hi @David405 you are doing good but I would like you to not rush things, you seem to be missing the TSEs correction, currently the page content is centered and the problem is from this style.

**Comment by Ispirett** [comment]
# Reviewing

File: style.css [hunk 5]
```diff
@@ -0,0 +1,150 @@
+body {
+  font-family: 'Cormorant', serif;
+}
+.container {
+    width: 76.5%;
+    margin-left:  200px;
+}
+
+.hero-image {
+    background-image: url("/images/your_verse_hero.jpg");
```
Comment (Ispirett) [inline style.css]: The reason the image is not showing up on our end.  You forgot to add `./` instead of  just `/`

Comment (David405) [reply style.css]: Hi @ispirett I really don't why the view appears like that on your end, aside the background image which I have fixed, everything else seems to be properly aligned on my end.

Comment (David405) [reply style.css]: Screenshot from 2019-08-09 15-42-07

Comment (David405) [reply style.css]: cropped

**Comment by samgaco** [comment]
### Hi @David405 :hand: 

### You've gone through a lot with this project, so congratulations on it :+1: It's approved :tada: 

This is how the [live-demo](https://rawcdn.githack.com/David405/BackgroundAndGradients/cc2fa1c87cd5c82e8c473c4988ad1915211b7f7a/index.html#) looks in my end.

It seems to maybe have something to do with the addition of white margins to the site.



And please notice, that you updated the readme file for the master branch, but not for the "dev" branch. Be careful with that when merging.

**[OPTIONAL]**  "watch the "Yout Verse"..." text image is appearing twice and the American flag icon is not showing, so you could try to address this.
**[SUGGESTION]**  _remember to review this section of the curriculum on[ git-flow ](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/)before moving on with your next project._

### Regards!, Keep up that diligent working methodology :muscle: . You can now submit the project completion form.

**Pull request merged**